components/mixins/profile/profile-mobile-plan-details.schema.json

```diff
@@ -12,7 +12,9 @@
   "meta:extensible": true,
   "meta:abstract": true,
   "meta:tags": { "industry": ["telecom"] },
-  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/profile"],
+  "meta:intendedToExtend": [
+    "https://ns.adobe.com/xdm/mixins/profile/profile-telecom-subscription"
```

Comment:
it should stay as profile, Add meta:extends to add properties from profile-telecom-subscription